### PR TITLE
feat: Thai handwriting font

### PR DESCRIPTION
**Pull request by peam1146**
### Description:
This pull request addresses the issue of missing font support for Thai Unicode characters in Excalidraw by introducing a fallback font named ~~SanamDeklen~~ Itim.

### Proposed Changes:
In this pull request, we propose adding a fallback font named ~~SanamDeklen~~ Itim to address the missing font support for Thai Unicode characters. This font is well-suited for Thai script and will enhance the readability and aesthetics of Thai text within Excalidraw.

### What I did:
- [x] The ~~SanamDeklen~~ Itim font files have been added to the project.
- [x] The font fallback mechanism has been updated to include ~~SanamDeklen~~ Itim for Thai Unicode characters.

### CODE REVIEW

**Comment by peam1146** [comment]
CleanShot 2566-09-07 at 14 42 47@2x

File: src/constants.ts [hunk 1]
```diff
@@ -99,6 +99,10 @@ export const FONT_FAMILY = {
   Cascadia: 3,
 };
 
+export const FALLBACK_FONT = {
```
Comment (DanielJGeiger) [inline src/constants.ts:102]:
```suggestion
export const FALLBACK_FONTS = {
```

File: src/utils.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import {
   CURSOR_TYPE,
   DEFAULT_VERSION,
   EVENT,
+  FALLBACK_FONT,
```
Comment (DanielJGeiger) [inline src/utils.ts:7]:
```suggestion
  FALLBACK_FONTS,
```

File: src/utils.ts [hunk 3]
```diff
@@ -92,6 +93,10 @@ export const getFontFamilyString = ({
 }) => {
   for (const [fontFamilyString, id] of Object.entries(FONT_FAMILY)) {
     if (id === fontFamily) {
+      const fallback = FALLBACK_FONT[fontFamily];
+      if (fallback) {
+        return `${fontFamilyString}, ${fallback}, ${WINDOWS_EMOJI_FALLBACK_FONT}`;
```
Comment (DanielJGeiger) [inline src/utils.ts:98]:
```suggestion
      const fallbacks = FALLBACK_FONTS[fontFamily];
      if (fallbacks) {
        return `${fontFamilyString}, ${fallbacks}, ${WINDOWS_EMOJI_FALLBACK_FONT}`;
```

**Review by DanielJGeiger — COMMENTED** [review]
I think your approach looks promising for #6679.  My suggested changes just make a couple variable names more multilingual.

cc @ad1992 @dwelle

**Comment by peam1146** [comment]
Do I have to do anything to fix to failed CI? @DanielJGeiger

**Comment by DanielJGeiger** [comment]
@peam1146 You can ignore the failed size-limit action.  For what it's worth, the action failed with a type error in `src/scene/Shape.ts`, which this PR doesn't modify.

**Review by DanielJGeiger — APPROVED** [review]

**Comment by peam1146** [comment]
Any update for this PR? @DanielJGeiger

**Comment by dwelle** [comment]
Hi. Adding new fonts/languages turned out non-trivial given all the factors at play (emoji support, server export, svg embedding and subsetting, text wrapping, hosting etc.) that until we're able to stabilize and automate the API, every new font needs to be handled carefully.

Currently we're working on CJK support (https://github.com/excalidraw/excalidraw/pull/8425). After that we can take a look at other fonts. As such, closing the PR for now, but feel free to open an issue if one does not exist already.

**Pull request closed without merging**